spreadsheet/formula/context.go

```diff
@@ -17,6 +17,24 @@ type Context interface {
 	// when evaluating cells that pull data from other sheets (e.g. ='Sheet 2'!A1)
 	Sheet(name string) Context
 
+	// GetFilename returns the full filename of the context's Workbook
+	GetFilename() string
+
+	// GetWidth returns a worksheet's column width
+	GetWidth(colIdx int) float64
+
+	// GetFormat returns a cell's format
+	GetFormat(cellRef string) string
+
+	// GetFormat returns cell's label prefix dependent on cell horizontal alignment
```

Comment:
GetLabelPrefix...
Each comment sentence should end with a dot also as per godoc convention. I.e.
https://golang.org/doc/effective_go.html#commentary
```
Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.
```

Reply:
fixed